chain/beacon/drand/drand.go

```diff
@@ -53,6 +53,9 @@ type DrandBeacon struct {
 	localCache *lru.Cache[uint64, *types.BeaconEntry]
 }
 
+// IsChained tells us whether this particular beacon operates in "chained mode". Prior to Drand
+// quicknet, beacons form a chain. After the introduction of quicknet, they do not so need to be
+// validated differently (See FIP-0063)
```

Comment:
differently?

Reply:
I can english..

changed this sentence to be a bit more clear, ptal